syntax/internal/stdlib/stdlib.go

```diff
@@ -75,9 +75,11 @@ var file = map[string]interface{}{
 }
 
 var encoding = map[string]interface{}{
-	"from_json":   jsonDecode,
-	"from_yaml":   yamlDecode,
-	"from_base64": base64Decode,
+	"from_json":    jsonDecode,
+	"from_yaml":    yamlDecode,
+	"from_base64":  base64Decode,
+	"to_base64":    base64Encode,
+	"to_URLbase64": base64URLEncode,
```

Comment:
Should there be an appropriate from_urlbase64?

Reply:
Have added the same makes sense since to have decode when we have the encode